### DISCUSSION TITLE
Media could not be loaded

### DISCUSSION THREAD

**Discussion by zeiktuvai** [stashapp/stash#3402] [Q&A] [answered]
Hello everyone, I deployed stash to a test server to see how it worked and got everything up and running and was able to view a video.

Now that I have deployed into my docker swarm it fails to play anything stating that the media could not be loaded (yes I used the same file that i did in test).

Everything is hosted on NFS and the container stores the database on the same share (in a different folder).  Stash scans the media folder, finds all the files and even creates thumbnails.  I just can't play anything.  And the docker logs don't indicate any issue either (just a constant stream of connecting to graphql)

Any assistance would be much appreciated, thank you.

**Comment by zeiktuvai** [comment, marked as answer]
So after joining the Discord and searching around more, I noticed an issue with the same occurrence but through Nginx.

I added `proxy_hide_header content-security-policy` to my nginx config and it is now working.  I looked at the nginx config in the guide and did not see that in there.  I am curious as to why this particular line causes an issue.

**Comment by billouetaudrey** [comment]
Hi, someone have the fix for Apache2 ? Thanks :)
